contracts/secret-storage/SecretStore.sol

```diff
@@ -516,23 +516,26 @@ contract SecretStore is
 
     function _slashEnclave(
         address _enclaveAddress,
-        uint256 _sizeLimit,
-        address _recipient,
-        uint256 _secretId
-    ) internal returns (uint256) {
-        uint256 totalComp = (secretStorage[_enclaveAddress].stakeAmount * SLASH_PERCENT_IN_BIPS) / SLASH_MAX_BIPS;
-        secretStorage[_enclaveAddress].stakeAmount -= totalComp;
+        uint256 _endTimestamp,
```

Comment:
can rename it to currentAliveOrDeadTimestamp

Reply:
update it at  other places too

Reply:
_currentCheckTimestamp seemed better